ui/pages/Project/components/edit-families-and-individuals/BulkEditForm.jsx

```diff
@@ -62,21 +67,21 @@ const FAM_UPLOAD_FORMATS = [].concat(BASE_UPLOAD_FORMATS)
 FAM_UPLOAD_FORMATS[1] = { ...FAM_UPLOAD_FORMATS[1], formatLinks: [...FAM_UPLOAD_FORMATS[1].formatLinks, { href: 'https://www.cog-genomics.org/plink2/formats#fam', linkExt: 'fam' }] }
 
 
-const BaseBulkContent = React.memo(({ actionDescription, details, project, name, requiredFields, optionalFields, uploadFormats, exportConfig, blankExportConfig }) =>
+export const BaseBulkContent = React.memo(({ actionDescription, details, project, name, requiredFields, optionalFields, uploadFormats, exportConfig, blankExportConfig }) =>
   <div>
     <Container>
       To {actionDescription}, upload a table in one of these formats:
       <StyledTable>
         <Table.Body>
           {uploadFormats.map(({ title, ext, formatLinks }) =>
             <TableRow key={title}>
-              <TableCell>
+              <TitleTableCell>
                 <BoldText>{title}</BoldText> ({formatLinks ? formatLinks.map(
                   ({ href, linkExt }, i) => <span key={linkExt}>{i > 0 && ' / '}<a href={href} target="_blank">.{linkExt}</a></span>)
                   : `.${ext}`})
-              </TableCell>
+              </TitleTableCell>
               <TableCell>
-                {ext &&
+                {ext && project ?
```

Comment:
Instead of changing this out with a ternary based on project, just makes this conditional on whether or not `exportConfig` is defined

```
{ext &&
<span>
    download &nbsp;
    {blankExportConfig &&
    <span>
      template: <FileLink data={blankExportConfig} ext={ext} linkContent="blank" /> {exportConfig &&or&nbsp;}
    </span>
    }
   {exportConfig &&
    <FileLink data={exportConfig} ext={ext} linkContent="current individuals" />
   }
</span>
}
```

Reply:
Sounds good.